### ISSUE TITLE
Should this work w/ Visual Studio 2019 Community Edition?

### ISSUE THREAD

**Issue by gthaker** [league1991/CodeAtlasVsix#16]
I have a project with  a single .cpp file. When I say "analyze"  a window pops up and seem to run thru various commands without errors (as far as I can see, it disappears quickly),  and my main window is  just dark. Nothing shows. I tried to play around, tried 2-3 times. Any hints welcome, tool sounds awesome.

**Comment by BelightWavre** [comment]
The same for me. Can't get it working. I also use Visual Studio 2019 Community 16.7.5 to develop Xamarin Forms projects

**Comment by league1991** [comment]
It should work for community edition. 

Please place the cursor on a function/class and press alt+f, you should see a disk appearing in the UI, then use alt+c/v to show more relations.

**Comment by gthaker** [comment]
As I said, my main window is just dark/blank. No code shows so I cant place cursor anywhere.....

**Comment by league1991** [comment]
No, just place it in VS text editor, the place you type your code. then press alt+g

**Comment by hwooo** [comment]
> No, just place it in VS text editor, the place you type your code. then press alt+g

It doesn't work. Nothing is happened after placing the cursor on text editor and pressing alt+g.
